server/fishtest/static/css/application.css

```diff
@@ -612,3 +612,7 @@ input[type="checkbox"],
   pointer-events: none;
   opacity: 0.5;
 }
+
+/* remove transparency of background */
+.alert-success-non-transparent {
+}
```

Comment:
Empty rule?

Reply:
If I understood correctly we can drop this rule. Let me know that I will push directly a commit.

Reply:
@dav1312 BTW fell free to open a PR to clean up the rule yourself :)